cedar-policy/CHANGELOG.md

```diff
@@ -20,12 +29,9 @@
   particular, this applies to the action context, so `context has attr` can now
```

Comment:
I don't think this is in the release branch

Reply:
Could you update the validator part of this PR as I'm not familiar with it? Thanks.